dist/stat_data.py

```diff
@@ -1223,6 +1223,8 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     # Reconciliation statistics
     ##########################################
+    RecStat('rec_max_internal_page_deltas', 'max deltas seen on internal page during reconciliation'),
```

Comment:
Can we make these both a connection stat and a data source stats? We can move these to conn_dsrc_stats